### ISSUE TITLE
Crash on inject

### ISSUE THREAD

**Issue by ThePrimeDev** [ArnoldasMk/Kali#61]
Crash on inject issue. Ubuntu 22.04, I downgraded to kernel 5.13.13 from 5.15, not helped, downloaded precompiled gdb version from a fuzion help comment w same issue, not worked.

**Comment by sudg6** [comment]
@ThePrimeDev Why did you downgrade your kernel? It's a game sided issue. Even Osiris fails to be injected after update of two days ago.

**Comment by shilowoof** [comment]
yes i have same exact problem pls help i want my funny purple skins back

**Comment by sudg6** [comment]
Everybody is addicted to the cheats!

---- wawa wrote ----


yes i have same exact problem pls help i want my funny purple skins back

**Comment by ThePrimeDev** [comment]
> @ThePrimeDev Why did you downgrade your kernel? It's a game sided issue. Even Osiris fails to be injected after update of two days ago.

Because of other tickets said that it will fix it. Also, I didnt really follow updates, so..
